src/code-templates/libraries/logger/index.ts

```diff
@@ -2,44 +2,43 @@ import { Logger, LoggerConfiguration } from './definition';
 import PinoLogger from './pino.logger';
 
 export class LoggerWrapper implements Logger {
-  private underlyingLogger: Logger | null = null;
+  #underlyingLogger: Logger | null = null;
```

Comment:
Used JavaScript privates as they are really private unlike TypeScript private

Reply:
I don't have a strong opinion here, just remember that sometimes 

readability > correctness